data/saleor.toml

```
@@ -0,0 +1,5 @@
+name = "Saleor"
+description = "CMS for e-commerce"
```

```suggestion
description = "A headless, GraphQL commerce platform delivering ultra-fast, dynamic, personalized shopping experiences."
```

what do you think about this for the description? gives users a little more information about the features of the CMS. i just pulled it from the Saleor readme.